ros_buildfarm/templates/snippet/install_python3.Dockerfile.em

```diff
@@ -1,4 +1,4 @@
-@[if os_name == 'ubuntu' and os_code_name in ['saucy', 'vivid', 'wily', 'xenial']]@
+@[if os_name == 'debian' or os_name == 'ubuntu' and os_code_name in ['saucy', 'vivid', 'wily', 'xenial']]@
```

Comment:
Use `in` for checking for multiple `os_name`s.

I assume the `os_code_name` check also needs to be updated.